src/simsopt/geo/curveobjectives.py

```diff
@@ -476,3 +476,53 @@ def dJ(self):
 @deprecated("`MinimumDistance` has been deprecated and will be removed. Please use `CurveCurveDistance` instead.")
 class MinimumDistance(CurveCurveDistance):
     pass
+
+
+class LinkingNumber(Optimizable):
+
+    def __init__(self, curves):
+        Optimizable.__init__(self, depends_on=curves)
+        self.curves = curves
+        r"""
+        Compute the Linking number of a set of curves (whether the curves 
+        are interlocked or not).
+
+        The value is 1 if the are interlocked, 0 if not.
+        
+        .. math::
```

Comment:
There are some issues with the docstring math in the rendered version. To view the rendered docs, go to the `docs` directory and run `make html`. Then open `docs/build/html/index.html` in a browser. I think `\cross` should be `\times`, and the quotes after `:math:` should be slanted (the key next to !).